dashboard/src/modules/components/AuthComponent/LoginForm.jsx

```diff
@@ -83,68 +85,70 @@ const LoginForm = () => {
   }, [checkOkButton, details]);
 
   return (
-    <Card>
-      <CardTitle>
-        <Back toPage="/auth" />
-        <LoginHeader title={"Login to your Pbench account"} />
-        <AlertGroup isLiveRegion>
-          {alerts.map(({ title, key }) => (
-            <Alert variant="danger" title={title} key={key} />
-          ))}
-        </AlertGroup>
-      </CardTitle>
-      <CardBody>
-        <Form>
-          <FormGroup label="Username" isRequired fieldId="username">
-            <TextInput
-              isRequired
-              type="text"
-              id="username"
-              name="username"
-              value={details.username}
-              onChange={handleUsernameChange}
-            />
-          </FormGroup>
-          <FormGroup label="Password" isRequired fieldId="password">
-            <div className="password-holder">
-              <PasswordTextInput
+    <>
+      <Card>
```

Comment:
What does the blank tag do here?  (Why isn't the `<Card>` sufficient on its own?)

Reply:
Updated